product-specifications/example-2.md

```diff
@@ -2,12 +2,20 @@
 
 This tool will help users identify animals by asking a series of questions and then provide a list of most likely animals based on the user's responses. This tool will be used by wildlife conservationists to identify animals they encounter.
 
-## User Flow:
+## User Flow
 
-1. Users answer a series of questions about the observed animal.
-2. Based on user inputs, the system will analyze potential matches.
-3. The system will display a list of animals that most closely match the user's responses.
+1. Users answer a series of questions about the observed animal:
+    - What color is the animal?
+    - Where was the animal observed?
+    - How large was the animal?
```

Comment:
wouldn't it be good if we asked how heavy the animal is?

Reply:
How would the user know how heavy the animal is? Maybe for bugs and small safe animals this could be easy?